src/utils/index.ts

```diff
@@ -0,0 +1,115 @@
+import { type Match, type Set } from "wasp/entities";
+type MatchWithSets = Match & {
```

Comment:
```suggestion
import { type Match, type Set } from "wasp/entities";

type MatchWithSets = Match & {
```

To keep it nicely formatted